pkg/controller.v1/pytorch/pytorchjob_controller_test.go

```diff
@@ -37,6 +37,7 @@ var _ = Describe("PyTorchJob controller", func() {
 		interval     = time.Millisecond * 250
 		expectedPort = int32(8080)
 	)
+	var nprocPerNode = "auto"
```

Comment:
Should we make the `defaultNprocPerNode` the global variable in `pkg/apis/kubeflow.org/v1/pytorch_defaults.go`, then use it here instead of declaring this?

Reply:
Done